### PR TITLE
Fixed the README.md

### PR DESCRIPTION
**Pull request by coder006**
Fixed README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Here is a quick example of plotting an AIA image:
 ```python
 >>> import sunpy.map
```
Comment (Cadair) [inline README.md]: you will also need to add a `import sunpy.data.sample` line here to make this example complete.

**Comment by coder006** [comment]
@Cadair one will aloso need to download the sample data first i guess. Should that also be added here?

**Comment by Cadair** [comment]
@coder006 yeah I suppose.

**Comment by coder006** [comment]
@Cadair why is this failing? Any idea?

**Comment by Cadair** [comment]
Looks like an error, I have restarted it.

**Comment by coder006** [comment]
@Cadair can this be merged now?

**Comment by Cadair** [comment]
:+1:

**Pull request merged**